packages/graphql/src/graphql/schema.graphql

```diff
@@ -144,7 +152,7 @@ enum SessionBlockStatus {
 
 enum SessionStatus {
   COMPLETED
-  PLANNED
+  PREPARED
```

Comment:
A session is now "PREPARED" after the initial creation (as having both "PLANNED" and "SCHEDULED" does not make sense). "SCHEDULED" -> might use this in the future to say that a session will automatically start at a predefined time, for example.